test/test_images/autoscale/autoscale.go

```diff
@@ -173,6 +173,7 @@ func handler(w http.ResponseWriter, r *http.Request) {
 		go func() {
 			defer wg.Done()
 			fmt.Fprint(w, bloat(mb))
+			time.Sleep(time.Second)
```

Comment:
What if we already have a delay of 1s or more?

Reply:
What do you mean? The sleep option? Each bloat/sleep/prime etc runs in parallel, so it won't have any unanticipated effects unless a bloat and a sleep < 1s occur in the same call and the test expects it to get back right away (which does not appear to be the case because everything passes)

Reply:
I guess. Do you think 1s is enough for things to be killed?

Reply:
It is for Cloud Run.